base/simplebook/ansible/roles/simplebook/templates/supervisor-simplebook.conf.j2

```diff
@@ -0,0 +1,24 @@
+[program:simplebookserver]
+directory={{ simplebook_install_directory }}/services/api
+user:vagrant
+command=pipenv run gunicorn --reload --bind 0.0.0.0:3333 wsgi:app
+autostart=true
+autorestart=false
+stdout_logfile=/var/log/simplebook.output.log
```

Comment:
We've been letting supervisor set the stdout file for us, so they all end up in /var/log/supervisor.  Do we think we should change that?  If so, we should probably put this stuff in a /var/log/torque.  Maybe we should do apache logs to there as well?

I think if we keep them different per supervisor service, it will make diagnosing issues harder

Reply:
Oh that sounds fine!  So long as the logs wind up somewhere -- I think I was just opting to be explicit so folks would know where to look.  There's value to that probably, but not at the expense of fragmentation.

I'll try taking this out to make sure it's behaving as I am interpreting your comments here; not advocating a change at this point.